discover/discover.ml

```diff
@@ -14,28 +14,34 @@ type arg =
   | T of string (* check type available *)
   | DEFINE of string (* define symbol prior to including header files (promoted to config) *)
   | Z of string (* define symbol to zero if not defined after the includes (promoted to config) *)
+  | SVC of string * string * string (* define symbol S to value V if condition C is met after the includes (promoted to config) *)
```

Comment:
nitpick : SVC sounds like service, can find better name? COND or IF (with reordering of condition as first arg)?